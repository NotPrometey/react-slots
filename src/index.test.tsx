import React from 'react';
import renderer from 'react-test-renderer';
import { Template } from './index';

describe('TemplateComponent', () => {
  it('itself render', () => {
    const component = renderer.create(
      <Template slot='test_name'>content</Template>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
