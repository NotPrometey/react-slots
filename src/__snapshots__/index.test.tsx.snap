// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TemplateComponent itself render 1`] = `"content"`;
